cocotb/share/makefiles/simulators/Makefile.verilator

```diff
@@ -29,7 +29,11 @@ ifeq ($(VERILATOR_SIM_DEBUG), 1)
 endif
 
 ifeq ($(VERILATOR_TRACE),1)
-  EXTRA_ARGS += --trace --trace-structs
+  WAVES = 1
```

Comment:
One additional action here is to add `VERILATOR_TRACE` to `Makefile.deprecations` if it isn't already.

Reply:
Could `--trace-fst` be skipped? Although it is a great compressed format, it is only supported by gtkwave for viewing and Verilator for generation, as far as I know. Sticking with VCD would make things easier I think.

Reply:
Surfer might support it?

Reply:
There is also another open-source project looking to bring an FST recording backend for other simulators but seems a bit far in the future. I just don't think it is ready to be the default option. But those are my 2 cents.

Reply:
Surfer supports FST, just like gtkwave. So for those people using open source tools, both the producer and the consumer side are covered well. FST is significantly better, both in terms of features (type representation for enums, for example), as well as in compression (which also means less data to write to disk and hence faster simulations). Anyone using open source tools is likely to want FST. VCD is really just a safe fallback for interop.